adcgen/sympy_objects.py

```diff
@@ -320,7 +320,7 @@ def eval(cls, i: Expr, j: Expr) -> Expr | None:  # type: ignore[override]
             return S.Zero
 
         spi, spj = i.space[0], j.space[0]
-        valid_spaces = ["o", "v", "g", "c"]
+        valid_spaces = ["o", "v", "g", "c", "a"]
```

Comment:
actually: How shouldn't a delta_{pP} evaluate to zero? Currently a delta_{general aux} would be valid (nonzero) and the aux index would be the preferred index in the method below.